projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -54,11 +54,22 @@
       display: flex;
       width: 100%;
       align-items: center;
-      padding-left: 8px;
 
       .trigger-label {
         @include body-2-medium();
         padding-left: 8px;
+        width: calc(100% - 32px - 38px);
```

Comment:
Is there a way to do this without calc and hardcoded widths?  What if we use `flex: 1 1 auto` ?

Reply:
Trigger label will always overflow container unless it has some fixed or calculated width. Without it won't show ellipses.
I have tried `flex: 1 1 auto` but it is not solving the issue.

It looks hardcoded but actually, it is assuming only 2 simple things
1. trigger label count is less than 999.
2. Icon on the right has a width of 32px.

and for the last part, it is being taken care of by `flex-grow: 1` on trigger-label

Reply:
okay. its fine.